pkg/apiserver/authentication/options/authenticate_options.go

```diff
@@ -60,5 +66,6 @@ func (options *AuthenticationOptions) AddFlags(fs *pflag.FlagSet, s *Authenticat
 	fs.DurationVar(&options.AuthenticateRateLimiterDuration, "authenticate-rate-limiter-duration", s.AuthenticateRateLimiterDuration, "")
 	fs.BoolVar(&options.MultipleLogin, "multiple-login", s.MultipleLogin, "Allow multiple login with the same account, disable means only one user can login at the same time.")
 	fs.StringVar(&options.JwtSecret, "jwt-secret", s.JwtSecret, "Secret to sign jwt token, must not be empty.")
-	fs.DurationVar(&options.OAuthOptions.AccessTokenMaxAge, "access-token-max-age", s.OAuthOptions.AccessTokenMaxAge, "AccessTokenMaxAgeSeconds  control the lifetime of access tokens, 0 means no expiration.")
+	fs.DurationVar(&options.OAuthOptions.AccessTokenMaxAge, "access-token-max-age", s.OAuthOptions.AccessTokenMaxAge, "AccessTokenMaxAgeSeconds control the lifetime of access tokens, 0 means no expiration.")
+	fs.DurationVar(&options.MaximumClockSkew, "maximum-clock-skew", s.MaximumClockSkew, "The maximum time difference between the system clocks of the ks-apiserver that issued a JWT and the ks-apiserver that verified the JWT.")
```

Comment:
`maximum-clock-skew` is not a straight forward name, I suggest `access-token-validity-max-toleration`

Reply:
I think `maximum-clock-skew` is a general option below `OAuthOptions`, option usage is also explained clearly.

https://docs.microsoft.com/en-us/dotnet/api/system.identitymodel.configuration.identityconfigurationelement.maximumclockskew?view=netframework-4.8
https://docs.oracle.com/cd/E19253-01/816-4557/setup-192/index.html
https://docs.sdl.com/801922/468896/sdl-tridion-docs-13-sp2/configuring-sso-in-livecontentsso-xml